### DISCUSSION TITLE
Error 14:45:46 in Lesson 13

### DISCUSSION THREAD

**Discussion by mohzourob** [smartcontractkit/full-blockchain-solidity-course-py#581] [General]
u can fix that error without editing ts config with uses JSON.stringfiy

`const networkName:string = chainId ? JSON.parse(JSON.stringify(helperConfig))[chainId] : "dev";`

**Comment by mohzourob** [comment]
@PatrickAlphaC

**Reply by PatrickAlphaC** [reply]
What's the question?

**Reply by mohzourob** [reply]
@PatrickAlphaC no question just a note.
I have finished ur course.
thx very much.
